docs/dashboard/configuration/channels.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Channels
+---
+
+This page allows you to define and manage your sales channels for your store. You can create multiple channels with different currencies.
+
+![Channels](../screenshots/channels-main.jpg)
+
+## How to
+
+### How to create a channel
+
+1. The first step is to click the _CREATE CHANNEL_ button to create a channel in the _Channels_ section of dashboard configuration.
+
+2. The second step is to fill out the following information:
```

Comment:
Should we provide information about that you can't change channel currency?

Reply:
Ok, I'll provide a note about it.